### DISCUSSION TITLE
BottomSheet over tabScreens & BackgroundColor?

### DISCUSSION THREAD

**Discussion by salahbm** [gorhom/react-native-bottom-sheet#1344] [Q&A]
Hi there : ) I have questions hope some can help ...
1. Is it possible to make bottomsheet over the bottom Tabs?
2. When I use btttom sheet I want that rest of the screen be 'rgba(0, 0, 0, 0.4)' color which I could do make it using Modal(but it was kinda slopy). Now with bottomsheet I did it with stateHooks however there is still a bit time difference.
How these can be solved? Thanks...

**Comment by ovidb** [comment]
I would really like to know the answer to this exact same question, as in how do you render the bottom sheet over the bottom Tabs

**Reply by dammyluup** [reply]
@salahbm @ovidb  i want to implement the exact behaviour you described,
i want the BottomSheet to go over the bottom nav. please any help ?

**Reply by salahbm** [reply]
Hey guys, I did by giving
backgroundColor: "#desiredOne"
containerStyle: zIndex:999

**Reply by dammyluup** [reply]
@salahbm thanks, wrapping my whole app with a BottomSheetModalProvider did the trick.
```wrap your NavigationContainer with the BottomSheetProvider or BottomSheetModalProvider```
